py/desispec/test/test_pipeline_run.py

```diff
@@ -4,145 +4,84 @@
 
 import os
 import unittest
-from uuid import uuid4
 import shutil
 import time
 import numpy as np
 
-# from desispec.pipeline.run import *
-# import desispec.io as io
+import subprocess as sp
 
-#- TODO: override log level to quiet down error messages that are supposed
-#- to be there from these tests
+from desispec.pipeline.common import *
+from desispec.pipeline.graph import *
+from desispec.pipeline.plan import *
+from desispec.pipeline.run import *
```

Comment:
Stylistically let's avoid using "import *" in the future since it makes it harder for people less familiar with the code to know which of the packages a given function/class came from.  In this case it is within some test code so it isn't that big a deal and I won't block the PR for it.